### ISSUE TITLE
.env sample file modification?

### ISSUE THREAD

**Issue by jungcw** [IBM/personal-wealth-portfolio-mgt-bot#7]
Thanks for providing good example journey.

Actually walked through of it and also tried running this on my local nodeJS environment... Turns out that provided .env.sample default value seems to be modified to make this working...

CRED_SIMULATED_INSTRUMENT_ANALYTICS_URL=https://fss-analytics.mybluemix.net/api/v1/scenario/instrument
==>
CRED_SIMULATED_INSTRUMENT_ANALYTICS_URL=https://fss-analytics.mybluemix.net/

URL_GET_PORTFOLIO_HOLDINGS=https://investment-portfolio.mybluemix.net/api/v1/portfolios
==>
URL_GET_PORTFOLIO_HOLDINGS=https://investment-portfolio.mybluemix.net/

**Comment by jungcw** [comment]
Tried Twilio portion as well. For my case, I used 2 Twilio free account phone numbers since it seems Twilio phone number doesn't accept SMS from my mobile phone number (Korea)

Anyway, managed to send SMS ("my portfolio?" as text message) via Twilio phone number, going through ngrok and can check that it hits my locahost:3000 nodeJS running instance. Can find following response being printed (added some debug printing statement). 

~~~
  bot:channel:twilio-sms response is: Your portfolio consists of 1500 shares of IBM, 5000 shares of GE, 5000 shares of F, 1800 shares of BAC +1ms
  bot:channel:twilio-sms <?xml version="1.0" encoding="UTF-8"?><Response><Message>Your portfolio consists of 1500 shares of IBM, 5000 shares of GE, 5000 shares of F, 1800 shares of BAC</Message></Response> 
~~~

How this above Http post call's response message (triggered by SMS incoming message webhook http post call) being handled? My expectation was somehow http response message being handled properly and eventually delivered back as SMS message to the initial SMS sender. But no luck...
Any advice/help would be appreciated.
